themes/instaplay/template.html

```diff
@@ -0,0 +1,147 @@
+<style>
+  :host {
+    --_primary-color: var(--media-primary-color, #fff);
```

Comment:
I'm assuming this is what's needed to change the default color but also still allow it to be overridden. 👍 We'll need to update other themes with this.